Лабораторная работа 2

Amizx2 opened the pull request:
![изображение](https://github.com/user-attachments/assets/c8b3facb-321a-4f6b-84c1-5f11c57e4a50)

Amizx2 committed:
Add files via upload

ZebraHead22 commented on test 2.py:
Подумайте, как лучше округлить, просто или в какую-то определенную сторону, чтобы денег точно хватило...

Amizx2 replied:
Добавил округление в большую сторону если число не целое.

Amizx2 committed:
Update test 2.py

Добавил округление в большую сторону если существуют цифры после точки. Реализовал с использованием знаний данных именно в этом курсе.
